tutor/src/models/task-plans/student.js

```diff
@@ -109,6 +109,19 @@ class StudentTaskPlans extends Map {
     }
   }
 
+  @computed get isTeacherWaitingForLatest() {
+    return Boolean(
+      this.course.primaryRole.isTeacherStudent && !this.isLatestPresent
+    );
+  }
+
+  @computed get isLatestPresent() {
+    if (this.course.primaryRole.isStudent) { return false; }
```

Comment:
This line seems odd. Is it just because students can't get the plans?

Reply:
hm yeah it does seem odd.  I'd put it there just so it wouldn't do anymore work in the case of a student but really there's not much more to do since for a student the teacher plans will be empty.  I'll remove it